### ISSUE TITLE
Allow spaces for the NFTs tokens names

### ISSUE THREAD

**Issue by gokaiorg** [ElvenTools/elven-tools-cli#29]
Hi! Is it possible to have a space in the token name? Thank you
```
Enter the name for the collection token (ex. MyName123). 
Avoid spaces and special characters
 … Test Collection Name
```

**Comment by juliancwirko** [comment]
Hey, space is not alphanumeric, so I think it isn't possible: https://docs.elrond.com/developers/nft-tokens/#parameters-format.

**Comment by gokaiorg** [comment]
I ask this because I see in the Elrond explorer for example here https://explorer.elrond.com/collections/AVIA-de368a, that they have a space in the NFT name.

In my metadatas I have a space in the name, but after deployment and issue collection, it is the collection name (without spaces) that seems to be used for display in the Elrond explorer.

<img width="851" alt="Capture d’écran 2022-04-11 à 10 24 09" src="https://user-images.githubusercontent.com/8545464/162661180-6fa06129-45fe-429f-831d-734c6fca4a1b.png">

**Comment by juliancwirko** [comment]
Oh, you're right. In elven tools, it is taken from the collection name, this is why it isn't possible, but this could be a good improvement, I think it could be added as an option when deploying the contract. I will leave this issue and mark it as an enhancement. Thanks for pointing this out.

**Comment by juliancwirko** [comment]
Corresponding SC issue: https://github.com/ElvenTools/elven-nft-minter-sc/issues/45

**Comment by gokaiorg** [comment]
> Oh, you're right. In elven tools, it is taken from the collection name, this is why it isn't possible, but this could be a good improvement, I think it could be added as an option when deploying the contract. I will leave this issue and mark it as an enhancement. Thanks for pointing this out.

I saw for example that in Isengard they have an `NFT Name` field and also a `Description` field, which might also be useful?

<img width="880" alt="Capture d’écran 2022-04-11 à 16 17 45" src="https://user-images.githubusercontent.com/8545464/162706987-eccc0b1f-4be4-4691-a0ce-f2d9b8f0300e.png">

**Comment by juliancwirko** [comment]
Yes, descriptions could also be added, but this is a custom field, probably kept in the 'attributes' field:
https://docs.elrond.com/developers/nft-tokens/#nftsft-fields

**Comment by juliancwirko** [comment]
@gokaiorg implemented in SC 1.6.0 and CLI 1.7.0
you can add name for the NFT when issuing the collection token/handler (it is optional, otherwise the NFT will take the collection name as it was previously)

**Comment by gokaiorg** [comment]
> @gokaiorg implemented in SC 1.6.0 and CLI 1.7.0
> you can add name for the NFT when issuing the collection token/handler (it is optional, otherwise the NFT will take the collection name as it was previously)

Will test tomorrow, thanks so much 🙏
